.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+
+name: buildx
+
+on:
```

Comment:
This seems like something that should run always to verify the build, and just not push if it’s not a tag?

Reply:
Sure! If want test ,may be fork and push a tag .
If want enable nighty version，first need to modify the Dockerfile and add ARG.

Reply:
I don't really want a nightly, so not pushing anything to docker hub, but I want the build steps in the repo to be exercised on commits to `main` and pull requests.

Reply:
I think so too, but in fact images built using the main branch do not work properly.

I have tried the following, but this requires a allowed version or debug mode:
```yml
on:
  pull_request:
    branches: main
  push:
    branches: main
  push:
    tags:
      - v*
```